spec/features/unauth_users/user_manages_cart_spec.rb

```diff
@@ -22,7 +22,7 @@
   it "allows users to remove things from their cart" do
     create_item_and_add_to_cart
     visit cart_path
-    click_link_or_button "Remove from Cart"
+    page.click_link('', :href => '/cart_items/1')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.